pybop/optimisers/pints_optimisers.py

```diff
@@ -18,21 +438,34 @@ class GradientDescent(pints.GradientDescent):
         Initial step size (default is 0.1).
     bounds : dict, optional
         Ignored by this optimiser, provided for API consistency.
+    **optimiser_kwargs : optional
+        Valid PINTS option keys and their values.
 
     See Also
     --------
     pints.GradientDescent : The PINTS implementation this class is based on.
     """
 
-    def __init__(self, x0, sigma0=0.1, bounds=None):
+    def __init__(self, x0=None, sigma0=0.1, bounds=None, **optimiser_kwargs):
```

Comment:
Should x0 be optional in these classes? The pints' class has it as a required argument. 

Also a good time to update the default sigma0 value for the GradientDescent class:


```suggestion
    def __init__(self, x0=None, sigma0=0.02, bounds=None, **optimiser_kwargs):
```